### PR TITLE
Progress info: Reset between syncs #4856

### PR DESCRIPTION
**Pull request by ckamm**

### CODE REVIEW

**Comment by guruz** [comment]
Also fixes #4832 ?

**Comment by ckamm** [comment]
I don't know. It shouldn't have affected the upload speed computation.

File: src/libsync/progressdispatcher.cpp [hunk 1]
```diff
@@ -127,9 +127,26 @@ void ProgressDispatcher::setProgressInfo(const QString& folder, const ProgressIn
...
+    _currentDiscoveredFolder.clear();
+    _sizeProgress = Progress();
+    _fileProgress = Progress();
+    _totalSizeOfCompletedJobs = 0;
+    _maxBytesPerSecond = 100000.0;
+    _maxFilesPerSecond = 2.0;
+    _updateEstimatesTimer.stop();
+}
+
+void ProgressInfo::start()
```
Comment (jturcotte) [inline src/libsync/progressdispatcher.cpp]: Reminder: rename to say what is starts

**Comment by ckamm** [comment]
@jturcotte Try again now. :)

**Pull request merged**